sentry/templates/deployment-vroom.yaml

```diff
@@ -0,0 +1,123 @@
+      annotations:
+        {{- if .Values.vroom.annotations }}
+{{ toYaml .Values.vroom.annotations | indent 8 }}
+        {{- end }}
+      labels:
+        app: {{ template "sentry.fullname" . }}
+        release: "{{ .Release.Name }}"
+        role: vroom
+        {{- if .Values.vroom.podLabels }}
+{{ toYaml .Values.vroom.podLabels | indent 8 }}
+        {{- end }}
+    spec:
+      affinity:
+      {{- if .Values.vroom.affinity }}
+{{ toYaml .Values.vroom.affinity | indent 8 }}
+      {{- end }}
+      {{- if .Values.vroom.nodeSelector }}
+      nodeSelector:
+{{ toYaml .Values.vroom.nodeSelector | indent 8 }}
+      {{- end }}
+      {{- if .Values.vroom.tolerations }}
+      tolerations:
+{{ toYaml .Values.vroom.tolerations | indent 8 }}
+      {{- end }}
+      {{- if .Values.images.vroom.imagePullSecrets }}
+      imagePullSecrets:
+{{ toYaml .Values.images.vroom.imagePullSecrets | indent 8 }}
+      {{- end }}
+      {{- if .Values.vroom.securityContext }}
+      securityContext:
+{{ toYaml .Values.vroom.securityContext | indent 8 }}
+      {{- end }}
+      containers:
+      - name: {{ .Chart.Name }}-vroom
+{{- if .Values.vroom.args }}
+        args:
+{{ toYaml .Values.vroom.args | indent 10 }}
+{{- end }}
+        image: "{{ template "vroom.image" . }}"
+        imagePullPolicy: {{ default "IfNotPresent" .Values.images.sentry.pullPolicy }}
+        ports:
+        - containerPort: {{ template "vroom.port" }}
+        env:
+        - name: VROOM_PORT
+          value: '{{ template "vroom.port" }}'
+{{- if .Values.vroom.env }}
+{{ toYaml .Values.vroom.env | indent 8 }}
+{{- end }}
+{{- if .Values.vroom.volumeMounts }}
+        volumeMounts:
+{{ toYaml .Values.vroom.volumeMounts | indent 10 }}
+{{- end }}
+{{/*        livenessProbe:*/}}
+{{/*          failureThreshold: {{ .Values.vroom.probeFailureThreshold }}*/}}
+{{/*          httpGet:*/}}
+{{/*            path: {{ template "relay.healthCheck.livenessRequestPath" }}*/}}
+{{/*            port: {{ template "relay.port" }}*/}}
+{{/*            scheme: HTTP*/}}
+{{/*          initialDelaySeconds: {{ .Values.vroom.probeInitialDelaySeconds }}*/}}
+{{/*          periodSeconds: {{ .Values.vroom.probePeriodSeconds }}*/}}
+{{/*          successThreshold: {{ .Values.vroom.probeSuccessThreshold }}*/}}
+{{/*          timeoutSeconds: {{ .Values.vroom.probeTimeoutSeconds }}*/}}
+{{/*        readinessProbe:*/}}
+{{/*          failureThreshold: {{ .Values.vroom.probeFailureThreshold }}*/}}
+{{/*          httpGet:*/}}
+{{/*            path: {{ template "relay.healthCheck.readinessRequestPath" }}*/}}
+{{/*            port: {{ template "relay.port" }}*/}}
+{{/*            scheme: HTTP*/}}
+{{/*          initialDelaySeconds: {{ .Values.vroom.probeInitialDelaySeconds }}*/}}
+{{/*          periodSeconds: {{ .Values.vroom.probePeriodSeconds }}*/}}
+{{/*          successThreshold: {{ .Values.vroom.probeSuccessThreshold }}*/}}
+{{/*          timeoutSeconds: {{ .Values.vroom.probeTimeoutSeconds }}*/}}
```

Comment:
Maybe it can be removed? Or add probes that will work

Reply:
removed commented lines, could not find any health check to use

Reply:
There is a health check on [`/health`](https://github.com/getsentry/vroom/blob/2ec56bbfbea46a0272413359e147c66116adce68/cmd/vroom/main.go#L148)